spec/views/hq/orphans/_form_spec.rb

```diff
@@ -0,0 +1,103 @@
+require 'rails_helper'
+require 'cgi'
+
+RSpec.describe "hq/orphans/_form.html.erb", type: :view do
```

Comment:
@NikitaAvvakumov, i see your point here. But I prefer to test this just in case you are missing something or some field is not fill correctly.